src/pip/_internal/commands/uninstall.py

```diff
@@ -64,7 +72,9 @@ def run(self, options, args):
                     '"pip help %(name)s")' % dict(name=self.name)
                 )
             for req in reqs_to_uninstall.values():
-                req.uninstall(
-                    auto_confirm=options.yes, verbose=options.verbose != 0
+                uninstall_pathset = req.uninstall(
```

Comment:
Modified this because we're reducing the attributes being set on `RequirementSet`.